open_academy/wizards/__init__.py

```diff
@@ -0,0 +1 @@
+from . import attendee_wizards
```

Comment:
```suggestion
from . import attendee_wizard
```

Change all it's references